containers/osbuild-composer/Dockerfile

```diff
@@ -43,7 +43,7 @@ RUN \
                         "--repofrompath=ephemeral0,${OSB_RPMREPO}" \
                         "--setopt=ephemeral0.gpgcheck=0" \
                         "--setopt=ephemeral0.priority=10" \
-                        install "osbuild-composer" \
+                        install "osbuild-composer-core" \
```

Comment:
The container-entrypoint supports `--builtin-worker`. This will not work with the worker not deployed in the container.

Reply:
Fair point, let's drop the support for the built-in worker?

Reply:
This really depends on what this container is used for. If it shall just provide a running composer instance, then yeah, it really should go. But if it should also support building images locally via weldr, then it should stay.

There is no real description of the purpose of this container, yet. So this really all depends on what this is meant to do.

Reply:
> This really depends on what this container is used for. If it shall just provide a running composer instance, then yeah, it really should go. But if it should also support building images locally via weldr, then it should stay.

I don't think running workers together with composer in the same container is worthwhile. The whole point of having a local worker (i.e., the reason why `osbuild-composer` depends on `osbuild-worker`) is to provide backwards compatibility for people using Image Builder on a local RHEL machine via the weldr API. While that's a fine use case, it's covered with the usual way of installing on RHEL: via the `osbuild-composer` package. It does not scale well to the service we're imagining, where we really want fleets of workers on different machines. I'd say there's no use-case to support it and would only make the container more complex than it needs to be.

> There is no real description of the purpose of this container, yet. So this really all depends on what this is meant to do.

Yeah, good point. Probably makes sense to add a README to `containers/osbuild-composer` stating what it is used for, or adding it to the header comment in `Dockerfile`.

Reply:
I dropped `--builtin-worker` from `entrypoint.py`.

Reply:
Some unused code is now left, which implements `builtin_worker`, but I guess it does no harm.

Reply:
I started cleaning it up, but realised I was more likely to break something, so figured I'd leave that for a future clean-up ;)